AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/manage-team/manage-team.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+    selector: 'app-manage-team',
+    templateUrl: './manage-team.component.html',
+    styleUrls: ['./manage-team.component.scss']
+})
+export class ManageTeamComponent implements OnInit {
+    constructor() {}
```

Comment:
can this be removed?

Reply:
if we should keep the ManageTeamComponent itself? If so I would because:
the skeleton code to verify a user can/cannot see this section already exists and this was to move it out of the work-allocations component
when the manage team stories start this will be where they would go

it's a refactor of existing template to make it clear where the boundaries are

Reply:
sorry I was referring to the empty constructor, not a deal breaker though. Can see you've removed it now :)